neurobooth_os/tasks/task.py

```diff
@@ -366,6 +368,103 @@ def gaze_contingency():
         pass
 
 
+class Color(Enum):
```

Comment:
EyeLinkColor to be more explicit. Would also make an IntEnum for added potential functionality.

Reply:
Also: Add class comment saying where these values come from? E.g., which page in a manual or a link to a forum thread, etc...

Reply:
Changed to EyelinkColor.
Added source documentation.
Python documentation says stick to Enum unless you need IntEnum for backwards compatibility or for strict typing of enum.value to int. Which, sure we do but anything outside of int 0-to-15 will anyway be rejected by the eyetracker. I think its ok to keep this as enum.